opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/services/EncryptionServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package de.adorsys.opba.protocol.facade.services;
+
+import com.google.crypto.tink.subtle.AesGcmJce;
+import de.adorsys.opba.protocol.api.services.EncryptionService;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import org.springframework.stereotype.Service;
+import org.springframework.web.context.annotation.RequestScope;
+
+import java.util.Base64;
+
+@Service
+@RequestScope
```

Comment:
While technically this is correct EncryptionService is service scoped, facade should not know anything about web environment. Suggest to remove Service and RequiredArgsConstructor and provide this as standalone java bean

Reply:
why remove RequiredArgsConstructor?

Reply:
@Service
@RequestScope
sorry, remove these

Reply:
done